config/settings/tickerplant.q

```diff
@@ -2,6 +2,7 @@
 
 \d .proc
 loadcommoncode:0b		// do not load common code
+tptype:`default
```

Comment:
is this required?

Reply:
There was talk of adding a `tptype` variable (either `default` or `segmented`) for different subscription logic depending on if stp or old tp is being used.  Before logic was using the `proc[proctype]` parameter passed into the subscribe function, but didn't want the logic to depend on the proctype.  I was just unsure of where/how to define the `tptype` variable in the old tp and stp to use in the subscription code, so put it here as a start.